packages/calcite-components/src/components/flow/flow.tsx

```diff
@@ -41,7 +41,11 @@ export class Flow implements LoadableComponent {
       ? lastItem.beforeBack
       : (): Promise<void> => Promise.resolve();
 
-    await beforeBack.call(lastItem);
+    try {
+      await beforeBack.call(lastItem);
+    } catch (_error) {
+      return;
```

Comment:
Can you add a `// back prevented` or similar one-liner comment to the catch block about this intentionally returning because back was canceled?

Reply:
done.